movement/watch_faces/complication/totp_face_lfs.c

```diff
@@ -41,23 +41,41 @@
 
 const char* TOTP_URI_START = "otpauth://totp/";
 
+/* The totp_record struct has two modes of operation:
```

Comment:
Not code you changed but above here:
Is it worth increasing MAX_TOTP_SECRET_SIZE now?
This code worked with very long 128 character secrets but I had to adjust this variable to 128 to get it to work. 48 seems quite low.

Also I don't see any reason for MAX_TOTP_RECORDS to be as low as 20 either?

Reply:
Will do. The reason to have a max of 20 is that we're still allocating this array every time (and to me 20 seems a lot to scroll through), but I'll push it to 30.